graphdatascience/query_runner/arrow_query_runner.py

```diff
@@ -49,10 +54,20 @@ def create(
                 encrypted,
                 disable_server_verification,
                 tls_root_certs,
+                arrow_version,
             )
         else:
             return fallback_query_runner
 
+    @staticmethod
+    def read_arrow_version(arrow_info: Series[Any]) -> ArrowVersion:
+        arrow_version = arrow_info.get("version", "alpha")
+        try:
+            arrow_version = ArrowVersion[arrow_version.upper()]
+            return arrow_version
+        except KeyError:
+            raise UnsupportedArrowVersion(arrow_version)
```

Comment:
asumming there is V2 on the server. But the server also supports V1 still. 
With this logic, we make it mandatory to update the client asap. Shouldnt we fallback to the latest known version and try if its supported?

Reply:
mh good point. maybe we should return a list of supported versions instead.

Reply:
done